docs/index.rst

```diff
@@ -11,32 +11,32 @@ MLRun Package Documentation
+MLRun is an open-source MLOps framework that offers an integrative approach to managing your machine-learning pipelines from early development through model development to full pipeline deployment in production.
+MLRun offers a convenient abstraction layer to a wide variety of technology stacks while empowering data engineers and data scientists to define the feature and models.
 
 Key Benefits
 ------------
 
 MLRun provides the following key benefits:
 
-- **Rapid deployment** of code to production pipelines
-- **Elastic scaling** for batch and real-time workloads
-- **Feature management** ingestion, preparation, and monitoring
-- **Works anywhere** in your local IDE, multi-cloud or on-prem
+- **Rapid deployment** of code to production pipelines
+- **Elastic scaling** of batch and real-time workloads
+- **Feature management** &mdash; ingestion, preparation, and monitoring
+- **Works anywhere** &mdash; your local IDE, multi-cloud, or on-prem
 
 Key Features
 --------------
 
 MLRun includes the following key features:
 
-- **Feature store** – Define and reuse features with a robust feature store that includes a highly flexible transformation framework.
-- **Elastic serverless runtimes** – Turn your Python code to composable functions, that can run at scale on Kubernetes, Dask and Horovod in a single command.
-- **Function marketplace** – Leverage a function marketplace to accelerate your model development process.
-- **Data ingestion & preparation** – Read and transform data from batch and online data stores.
-- **Model training & testing** – Train models at scale with automated testing functions.
-- **Real-time Data & model Pipeline** – Deploy real-time pipelines for data collection, model serving and monitoring.
-- **Data & model Monitoring** – Automate model monitoring and drift detection.
-- **Central data & metadata management** – Log all data, models and artifacts and track all code execution.
-
+- **Feature store** &mdash; define and reuse features with a robust feature store that includes a highly flexible transformation framework.
+- **Elastic serverless runtimes** &mdash; use a single command to turn your Python code into composable functions that can run at scale on Kubernetes, Dask, and Horovod.
+- **Function marketplace** &mdash; leverage a function marketplace to accelerate your model-development process.
+- **Data ingestion & preparation** &mdash; read and transform data from batch and online data stores.
+- **Model training & testing** &mdash; train models at scale with automated testing functions.
+- **Real-time Data & model Pipeline** &mdash; deploy real-time pipelines for data collection and model serving and monitoring.
+- **Data & model Monitoring** &mdash; automate model monitoring and drift detection.
+- **Central data & metadata management** &mdash; log all data, models and artifacts and track all code execution.
```

Comment:
Last I checked &mdash; doesn't work in rst. That's why I put an actual mdash character

Reply:
@gilad-shaham OK, I reverted to using an em-dash Unicode character (—) instead of `&mdash;` in **docs/index.rst**.

Reply:
Gilad approved the PR. => I'm resolving this conversation.